### DISCUSSION TITLE
Is there a status page?

### DISCUSSION THREAD

**Discussion by binwiederhier** [healthchecks/healthchecks#589] [Q&A]
Thank you very much for for making this tool, and for making it free.

This may be a stupid question, but I recently added a badge to my project (https://github.com/binwiederhier/ntfy), and I was kinda hoping there'd be also a status page, just like https://status.healthchecks.io/en/

Is that something that healthchecks.io offers. And if not, how would I go about integrating healthchecks.io into a status page provider? How did you make yours?

**Comment by niklas2810** [comment]
As far as I am concerned, healthchecks does not support publicly accessible status pages at the moment. However, it exposes a scraping site for prometheus: https://healthchecks.io/docs/configuring_prometheus/

Keep in mind that it exposes *all* your checks. You could build a wrapper around the endpoint or just use Grafana (this is what I'm doing) to get a more refined dashboard.

**Comment by binwiederhier** [comment]
Thanks for the answer. I was able to hook it into statuspage.io using the e-mail integration. It was dead-simple to set up: https://ntfy.statuspage.io -- Might be worth mentioning in the docs. Thanks again!

![211209 13-42-45 Selection 001](https://user-images.githubusercontent.com/664597/145457211-4929bde7-0df3-4903-aa2c-2eb82c0a84bb.png)

![image](https://user-images.githubusercontent.com/664597/145457355-1bfa1ac3-fe0d-40b4-802a-e9460b78b909.png)

**Comment by cuu508** [comment]
Fully featured status pages (like the ones from statuspage.io) are not in scope for Healthchecks. In a status page I expect the ability to post incidents, postmortems, planned maintenances. Have support for active uptime checks, custom metrics. Allow visitors to subscribe for updates. Healthchecks doesn't have (and I have no plans for) any of that.

There is the already mentioned Prometheus endpoint, there is also [API](https://healthchecks.io/docs/api/) that you can use to build dashboards. Examples:

https://github.com/healthchecks/dashboard
https://github.com/nicoandrade/healthchecks-front

For integration with statuspage.io, @binwiederhier, I think the email method you used is ideal – as long as you don't have too many components to update.
